src/3rd_party/apache-log4cxx/helpers.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+
+#
+#   Helper functions for apache-log4cxx versioning
+#
+
+DEBUG() {
+    echo "$@"
+}
+
+AppendCommitHash() {
+    LOG4CXX_SOURCE_DIRECTORY=$1
+    LOG4CXX_LIBS_DIRECTORY=$2
+    COMMIT_HASH_FILE=/tmp/commit_hash
+    cd $LOG4CXX_SOURCE_DIRECTORY && git log --pretty=format:%H -1 . 1>$COMMIT_HASH_FILE 2>/dev/null
```

Comment:
@mkorniichuk do we rely need to mute errors?

Reply:
@LuxoftAKutsan fixed within bf0ee058043a3c5988f9ca2f28ecf5c31183842f